ppr-ui/src/assets/styles/base.scss

```diff
@@ -845,4 +848,14 @@ tr.registration-row.draft-registration-row.added-reg-effect td.actions-cell,
 }
 .theme--light.v-application code {
   white-space: break-spaces;
+}
+
+// Section H2 Counters
+.increment-sections {
+  counter-reset: section 0;
+
+  section h2:before {
+    counter-increment: section;
+    content: counters(section, ".") ". ";
+  }
 }
```

Comment:
We increment sections all over the place, with this class we no longer need it locally in the components. 
Just wrap the sections you want to 'count' and append the 'counter' too, with this class `increment-sections`.

This allows us to have our `h2`s other places in the same view without being part of the count.